plugins/dell-kestrel/fu-dell-kestrel-ec.c

```diff
@@ -413,23 +430,36 @@ fu_dell_kestrel_ec_is_dock_ready4update(FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_dell_kestrel_ec_is_new_ownership_cmd(FuDellKestrelEc *self)
+{
+	FuDevice *device = FU_DEVICE(self);
+	const gchar *version = fu_device_get_version(device);
+	FwupdVersionFormat fmt = fu_device_get_version_format(device);
+
+	if (fu_version_compare(version, "01.00.00.00", fmt) >= 0)
+		return (fu_version_compare(version, "01.00.05.02", fmt) >= 0);
```

Comment:
i think it would be obvious to have `if (fu_version_compare() && fu_version_compare()) return TRUE here.

Reply:
separated the condition checks.